admin/app.py

```diff
@@ -215,20 +227,20 @@ def uploadFiles():
 @app.route('/book-storage')
 @login_required
 def book_storage():
-    books = Book.objects()
+    books = Book.objects()[:10]
```

Comment:
Why do we limit objects that we show in the book storage ?

Reply:
Because we have too many books to show that all on page, it's a temporary solution in that task